.storybook/themes/commerce.scss

```diff
@@ -209,15 +209,21 @@
 	$tokens: (
 		default: (
 			components: (
+				attribution: (
+					color: var(--text-color),
+					font-size: var(--body-font-size-small),
+					font-weight: var(--global-font-weight-7),
```

Comment:
```suggestion
					font-weight: var(--body-font-weight-small),
```

looks like this should be 400 based on mocks. this goes with the alias 400 for font weight small and font-size small

Reply:
<img width="1094" alt="Screen Shot 2022-05-06 at 10 22 05" src="https://user-images.githubusercontent.com/5950956/167163136-f1c45867-b4bc-40ed-838f-d5d04052ca7b.png">

Reply:
@JackHowa : Thanks, corrected! And no worry about the size of change. I want to be on par w/ Design specs 100%!